docs/local-testnet.rst

```diff
@@ -0,0 +1,83 @@
+Local Testnet
+=============
+
+This tutorial demonstrates the basics of setting up a gaia
```

Comment:
Can we add a comment as to the first tutorial in staking-module.rst - we should position the reader in each tutorial... unless that is already kind of implied via the UX of readthedocs?

Reply:
yeah, it's implied in the UX but I'll add some links where appropriate.

Reply:
that's fine then